go/internal/ocxcli/doctor_coordinator.go

```
@@ -0,0 +1,278 @@
+	}
+	before, err := os.Lstat(path)
+	if os.IsNotExist(err) {
+		return DoctorCoordinatorDiagnostic{Kind: "absent", Path: path}
+	}
+	if err != nil {
+		return DoctorCoordinatorDiagnostic{Kind: "unsafe", Path: path, Reason: "the coordinator file cannot be inspected"}
+	}
+	if !before.Mode().IsRegular() || before.Mode()&os.ModeSymlink != 0 || !doctorOwnedByCurrentUser(before) || before.Mode().Perm() != 0o600 {
+		return DoctorCoordinatorDiagnostic{Kind: "unsafe", Path: path, Reason: "the coordinator file has unsafe ownership or permissions"}
+	}
+	real, err := filepath.EvalSymlinks(path)
```

golangci-lint (2.13.2)

[error] variable real has same name as predeclared identifier

(predeclared)